src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -16,154 +17,176 @@ import {
 
 import { User } from "types/User";
 
-// @debt remove styled-components in favour of using our standard scss patterns
-import * as S from "./UserProfilePicture.styles";
 import "./UserProfilePicture.scss";
 
+const randomAvatarUrl = (id: string) =>
+  "/avatars/" +
+  RANDOM_AVATARS[Math.floor(id?.charCodeAt(0) % RANDOM_AVATARS.length)];
+
+// TODO: refactor this to accept named props instead of positional args
+const avatarUrl = (
+  id: string,
+  anonMode?: boolean,
+  pictureUrl?: string,
+  miniAvatars?: boolean
+): string => {
```

Comment:
[TODO] refactor this to accept named props instead of positional args

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1323/commits/9ae02ba722c2f12213e8670471155b083d1b2c51